src/material/slider/slider.html

```diff
@@ -2,21 +2,43 @@
 <ng-content></ng-content>
 
 <!-- Track -->
-<div class="mdc-slider__track">
+<div class="mdc-slider__track" [style.width]="_trackWidthStyle" [style.left]="_trackLeftStyle">
```

Comment:
I think when we do this via bindings we have to trigger change detection every time we want them updated. Why not just set them directly in the JS? e.g. `trackEl.style.width = '10px'` (same for these other constantly changing properties)